src/languages/en.ts

```diff
@@ -2241,7 +2264,7 @@ export default {
             },
             enterCredentials: 'Enter your credentials',
             connections: {
-                syncStageName: (stage: PolicyConnectionSyncStage) => {
+                syncStageName: ({stage}: {stage: PolicyConnectionSyncStage}) => {
```

Comment:
Using the PolicyConnectionSyncStage  here, instead, we should define a new type in`src/languages/types.ts` file:

```js
 type SyncStageNameParams = {
    stage: PolicyConnectionSyncStage
}
```
and then use like this:

```diff
- syncStageName: ({stage}: {stage: PolicyConnectionSyncStage}) => {
+ syncStageName: ({stage}: PolicyConnectionSyncStage) => {
```

Same changes for es.ts and